### ISSUE TITLE
Unable to debug python files (only on remote server): Timed out waiting for debuggee to spawn

### ISSUE THREAD

**Issue by srinidhiPY** [microsoft/debugpy#1139]
Hi,

The VS-Code debugger stopped working only on remote server, but it works on local system. It always shows an error "Timed out waiting for debuggee to spawn". Please, refer to the attached screenshot. I have tried downgrading the vscode-python but same issue still persists.

The debug config file (launch.json) file content is below and this config file was working earlier:
{
// Use IntelliSense to learn about possible attributes.
// Hover to view descriptions of existing attributes.
// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
"version": "0.2.0",
"configurations": [
{
"name": "Python: CLI",
"type": "python",
"request": "launch",
"program": "${file}",
"console": "integratedTerminal",
"justMyCode": true,
"args": [
"+simulations.plant_section=1",
"+simulations.plant_section.data='current'"]
}
]
}

The VS-code and Debugpy version details:
Version: 1.73.1 (user setup)
Commit: https://github.com/microsoft/vscode/commit/6261075646f055b99068d3688932416f2346dd3b
Date: 2022-11-09T04:27:29.066Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19044
Sandboxed: No
Debugpy version: 1.6.3
Python 3.9.15

Please, I request you to help me with this issue. Thanks for your time and consideration.
![debugger_timed_out](https://user-images.githubusercontent.com/16255955/205229988-6268aaf4-f59f-4ec5-8771-1acd6460cff5.jpg)

**Comment by int19h** [comment]
Judging by the log, it looks like it's trying to run Python from a conda env, but I don't see conda activation in the terminal. If you manually do `conda activate` before trying to run, does it help?

**Comment by srinidhiPY** [comment]
![debug_error](https://user-images.githubusercontent.com/16255955/205860427-f3414531-18e5-4f66-84cd-98b1099e0e20.jpg)

Hi, Thanks for the reply. Even by activating the environment (in this case, **csm_or** is the env name) doesn't help. I get the same error message. Please, find the attached screenshot. Please, kindly help me this.

**Comment by int19h** [comment]
Let's see if anything useful can be gleaned from the log. If you add `"logToFile": true` to your config in launch.json, it should produce a bunch of files named `debugpy*.log` in the directory where the Python extension is installed - `~/.vscode-server/extensions/ms-python.python-2022.18.2`. Could you share those?

**Comment by srinidhiPY** [comment]
[ms-python.python-2022.18.2.zip](https://github.com/microsoft/debugpy/files/10172843/ms-python.python-2022.18.2.zip)

Hi, 

I added the **"logToFile": true** in my config in launch.json. But, I don't see any files by name **debugpy*.log** in the **ms-python.python-2022.18.2** directory . However, I have attached the complete folder for your reference.

Also, I see a strange behavior now that I could able to debug python files now, but sometimes it fails randomly and throws me the previous error message ("Timed out waiting for debuggee to spawn"). This behavior is only when I debug on a remote server. Any suggestions?

Thanks for your kind help.
